library/src/scripts/content/translationGrid/TranslationLanguageHandler.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from "react";
+import classNames from "classnames";
+import { TranslationGrid } from "@library/content/translationGrid/TranslationGrid";
+import { ITranslationGrid } from "./TranslationGrid";
+import { panelListClasses } from "@library/layout/panelListStyles";
+import { translationGridClasses } from "@library/content/translationGrid/TranslationGridStyles";
+import LanguagesDropDown from "@library/layout/LanguagesDropDown";
+import { AlertIcon } from "@library/icons/common";
+import { useUniqueID } from "@library/utility/idUtils";
+import SelectBox, { ISelectBoxItem } from "@library/forms/select/SelectBox";
+import { useLocaleInfo, LocaleDisplayer, ILocale, loadLocales } from "@vanilla/i18n";
+import { ILoadable, LoadStatus } from "@library/@types/api/core";
+import { ToolTip, ToolTipIcon } from "@library/toolTip/ToolTip";
+import DateTime from "@library/content/DateTime";
+import Translate from "@library/content/Translate";
+import { ILanguageItem } from "@library/layout/LanguagesDropDown";
+
+export interface ITranslation {
+    id: string;
+    source: string;
+    translation: string;
+    multiLine?: boolean; // We'll default to a textarea, that looks like a single line, but it would be much better to know.
+    maxLength?: number; // Please add maximum character counts where possible.
+}
+
+export interface ITranslationDummy {
+    // for storybook
+    key: string;
+    locale: string;
+    source: string;
+    translation: string;
+    translationStatus: string;
+    multiLine?: boolean;
+    maxLength?: number;
+}
+export interface ITranslationLanguageHandler {
+    data: ITranslation[];
+    inScrollingContainer?: boolean;
+    otherLanguages: ILanguageItem[];
+    i18nLocales: ILocale[];
```

Comment:
Why are we passing this as a prop instead of using the `useLocaleInfo()` hook?

Reply:
We're also taking `otherLanguages` here, which we shouldn't need at the same time. We should be able to use the locale info to make our items.

Reply:
Done.
Removed hardcode data and use useLocaleInfo() to get locales